README.md

```diff
@@ -1 +1,136 @@
-# module-sap
\ No newline at end of file
+# module-sap
+
+## Package overview
+
+Ballerina SAP Client Endpoint is used to connect Ballerina with SAP. With this SAO Client Endpoint, Ballerina can act as SAP Consumers and SAP Producers.
```

Comment:
I believe we should reword this since we have both a client and a listener in this module.

Basically we have to say that the Ballerina SAP Client acts as a SAP producer while the Ballerina SAP Listener acts as a SAP Consumer.

IMO, we don't have to use the "endpoint" word here.